### PR TITLE
AI Face SDK update 

### PR DESCRIPTION
**Pull request by Han-msft**
- Added support for the Large Face List and Large Person Group
- Added support for latest Detect Liveness Session API
- Change the default service API version to `v1.2-preview.1`.

### CODE REVIEW

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    {
+        /// <summary> Initializes a new instance of FaceClient. </summary>
+        /// <param name="endpoint">
+        /// Supported Cognitive Services endpoints (protocol and hostname, for example:
+        /// https://{resource-name}.cognitiveservices.azure.com).
+        /// </param>
+        /// <param name="largeFaceListId"> Valid character is letter in lower case or digit or '-' or '_', maximum length is 64. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LargeFaceListClient(string largeFaceListId, Uri endpoint, AzureKeyCredential credential) : this(largeFaceListId, endpoint, credential, new AzureAIVisionFaceClientOptions())
```
Comment (chunyu3) [inline sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs:26]: Put `largeFaceListId` parameter behind the endpoint and credential 
```suggestion
        public LargeFaceListClient(Uri endpoint, AzureKeyCredential credential, string largeFaceListId) : this(endpoint, credential, largeFaceListId, new AzureAIVisionFaceClientOptions())
```

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+        /// <summary> Initializes a new instance of FaceClient. </summary>
+        /// <param name="endpoint">
+        /// <param name="largeFaceListId"> Valid character is letter in lower case or digit or '-' or '_', maximum length is 64. </param>
+        /// Supported Cognitive Services endpoints (protocol and hostname, for example:
+        /// https://{resource-name}.cognitiveservices.azure.com).
+        /// </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LargeFaceListClient(string largeFaceListId, Uri endpoint, TokenCredential credential) : this(largeFaceListId, endpoint, credential, new AzureAIVisionFaceClientOptions())
```
Comment (chunyu3) [inline sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs:38]: same as above
```suggestion
        public LargeFaceListClient(Uri endpoint, TokenCredential credential, string largeFaceListId, ) : this(endpoint, credential, largeFaceListId, new AzureAIVisionFaceClientOptions())
```

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        /// <summary> Initializes a new instance of LargeFaceListClient. </summary>
+        /// <param name="endpoint">
+        /// Supported Cognitive Services endpoints (protocol and hostname, for example:
+        /// https://{resource-name}.cognitiveservices.azure.com).
+        /// </param>
+        /// <param name="largeFaceListId"> Valid character is letter in lower case or digit or '-' or '_', maximum length is 64. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LargeFaceListClient(string largeFaceListId, Uri endpoint, AzureKeyCredential credential, AzureAIVisionFaceClientOptions options)
```
Comment (chunyu3) [inline sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs:51]: same as above
```suggestion
        public LargeFaceListClient(Uri endpoint, AzureKeyCredential credential, string largeFaceListId, AzureAIVisionFaceClientOptions options)
```

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargePersonGroupClient.cs [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+    {
+        /// <summary> Initializes a new instance of FaceClient. </summary>
+        /// <param name="endpoint">
+        /// Supported Cognitive Services endpoints (protocol and hostname, for example:
+        /// https://{resource-name}.cognitiveservices.azure.com).
+        /// </param>
+        /// <param name="largePersonGroupId"> ID of the container. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LargePersonGroupClient(string largePersonGroupId, Uri endpoint, AzureKeyCredential credential) : this(largePersonGroupId, endpoint, credential, new AzureAIVisionFaceClientOptions())
```
Comment (chunyu3) [inline sdk/face/Azure.AI.Vision.Face/src/Custom/LargePersonGroupClient.cs:26]: same change as `LargeFaceListClient` customized construtor

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs (hunk 1 above)
Comment (Han-msft) [reply sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs]: Updated

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs (hunk 2 above)
Comment (Han-msft) [reply sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs]: Updated

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs (hunk 3 above)
Comment (Han-msft) [reply sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs]: Updated

File: sdk/face/Azure.AI.Vision.Face/src/Custom/LargePersonGroupClient.cs (hunk 4 above)
Comment (Han-msft) [reply sdk/face/Azure.AI.Vision.Face/src/Custom/LargePersonGroupClient.cs]: Updated

**Review by chunyu3 — APPROVED** [review]

File: sdk/face/Azure.AI.Vision.Face/README.md [hunk 5]
```diff
@@ -97,18 +98,40 @@ AzureKeyCredential credential = new AzureKeyCredential("<your apiKey>");
 var client = new FaceClient(endpoint, credential);
 ```
 
+### Specifiy the service version
+
```
Comment (chunyu3) [inline sdk/face/Azure.AI.Vision.Face/README.md:102]: Please use the apiversion document template.

**Review by ShaoAnLin — APPROVED** [review]

**Comment by Han-msft** [comment]
Got approval for beta2 release from Krzysztof in email thread. Merging the PR.

**Pull request merged**